### PR TITLE
Youtube new branch pull request

### PR DESCRIPTION
**Pull request by Habtamu210**
Personal review for my project includes:
- HTML files
- Styling files

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Habtamu210 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid

You have submitted a project, that wasn't reviewed at all. Kindly make sure you review your code using the GitHub inline comments as shown in [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) lesson and after you can ask for a review.

- [ ] This project's goal is to learn how to use GitHub features for reviews and how to communicate professionally.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style/general.css [hunk 1]
```diff
@@ -6,8 +6,8 @@ p {
 
 body {
   margin: 0;
-  margin-top: 80px;
-  padding-left: 96px;
```
Comment (Habtamu210) [inline style/general.css:10]: I think it would be better to use margin and padding for specific elements inside the body !

File: youtube.html [hunk 2]
```diff
@@ -87,7 +87,7 @@
             &commat;Arts Tv World |&commat;Donkey Tube
           </p>
           <p class="video-status">
-            67,738 viewsAug 1, 2022
+            67,738 viewsAug 7, 2022
```
Comment (Habtamu210) [inline youtube.html:90]: I think you have made changes here please correct it also!

File: youtube.html [hunk 3]
```diff
@@ -87,7 +87,7 @@
...
           </p>
         </div>
```
Comment (Habtamu210) [inline youtube.html:93]: please remove unnecessary new line

**Review by Habtamu210 — COMMENTED** [review]
# Required changes ♻️ 
please check my comments and make required changes!

Submit for re-review after you have made the changes
Thank you!

**Review by Olamarx — APPROVED** [review]
Hi @Habtamu210, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of GitHub review features ✔️ 
- Correct grammatical composition ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**